script.js

```diff
@@ -186,3 +186,47 @@ form.addEventListener('submit', (event) => {
     form.submit();
   }
 });
+
+// Form Input Local storage
+function storageAvailable(type) {
+  let storage;
+  try {
+    storage = window[type];
+    const x = '__storage_test__';
+    storage.setItem(x, x);
+    storage.removeItem(x);
+    return true;
+  } catch (e) {
+    return (
+      e instanceof DOMException
+      && (e.code === 22
+        || e.code === 1014
+        || e.name === 'QuotaExceededError'
+        || e.name === 'NS_ERROR_DOM_QUOTA_REACHED')
+      && storage
+      && storage.length !== 0
+    );
+  }
+}
+
+function populateStorage() {
+  localStorage.setItem('name', document.getElementById('name').value);
+  localStorage.setItem('email', document.getElementById('email').value);
+  localStorage.setItem('message', document.getElementById('message').value);
+}
+
+function fillForm() {
+  document.getElementById('name').value = localStorage.getItem('name');
+  document.getElementById('email').value = localStorage.getItem('email');
+  document.getElementById('message').value = localStorage.getItem('message');
+}
+
+if (storageAvailable('localStorage')) {
+  if (localStorage.getItem('name')) {
+    fillForm();
+  }
+
+  form.addEventListener('submit', () => {
+    populateStorage();
+  });
```

Comment:
- [ ] According to requirements, after filling the form, and not submitting it, the entered values should be saved in the browser. But in your case, I noticed that you only save to `localStorage` when the submit button is clicked. Please rewrite this functionality to fulfill the requirement 🙏